assembly/src/main/assembly/unix.xml

```diff
@@ -137,13 +137,13 @@
             </includes>
             <useTransitiveDependencies>false</useTransitiveDependencies>
         </dependencySet>
-        <dependencySet>
+        <!--dependencySet>
```

Comment:
These changes look like they're working, but still waiting for validation. The "proper" commit would remove these rather than commenting them out